themes/src/main/resources/theme/keycloak.v2/account/src/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "keycloak-preview",
+  "name": "keycloak.v2",
```

Comment:
This comment is unrelated to this line.

There seems to be something wrong with the dist. It's ~50MB larger. I think the [resources filtering](https://github.com/keycloak/keycloak/blob/734ffb8f4212a947297935b0542f9a16fd02098f/themes/pom.xml#L85-L130) doesn't work anymore.

Reply:
Good catch.  I'll check it out.

Reply:
> This comment is unrelated to this line.
> 
> There seems to be something wrong with the dist. It's ~50MB larger. I think the [resources filtering](https://github.com/keycloak/keycloak/blob/734ffb8f4212a947297935b0542f9a16fd02098f/themes/pom.xml#L85-L130) doesn't work anymore.

Should be fixed now.